Bottom: 02f38ba18a823320de538b8842b0c793bb84892f
Top:    d5d0ee3e61125c5b14a2d3061e3a166d7e4bfb88
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-06 22:50:44 +1300

Refresh of define-stream-result

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index 3acb3d8..38428ac 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -352,9 +352,9 @@ class StreamResult(object):
             file. This can just be a single chunk of the file - emitting
             another file event with more later. Must be None unleses a
             file_name is supplied.
-        :param eof: This chunk is the last chunk of the file, any additional
-            chunks with the same name should be treated as an error and 
-            discarded. Ignored unless file_name has been supplied.
+        :param eof: True if this chunk is the last chunk of the file, any
+            additional chunks with the same name should be treated as an error
+            and discarded. Ignored unless file_name has been supplied.
         :param mime_type: An optional MIME type for the file. stdout and
             stderr will generally be "text/plain; charset=utf8". If None,
             defaults to application/octet-stream. Ignores unless file_name
